### ISSUE TITLE
Namespace transform breaking non-Prisma imports

### ISSUE THREAD

**Issue by pizzafox** [prisma/codemods#6]
```diff
diff --git a/index.js b/index.js
index 011d26b..b05f60b 100644
--- a/index.js
+++ b/index.js
@@ -1,5 +1,5 @@
-import { Person as LocalPerson } from "./local-file";
-import { Person } from "@prisma/client";
+import { Prisma.Person as LocalPerson } from "./local-file";
+import { Prisma } from "@prisma/client";
 
 /** @type {Person} */
 const person = new LocalPerson();
```

Reproduction [here](https://github.com/pizzafox/prisma-codemods-issue-6) and in this ZIP file [prisma-codemods-issue-6-main.zip](https://github.com/prisma/codemods/files/5592177/prisma-codemods-issue-6-main.zip).

**Comment by williamluke4** [comment]
Hey @pizzafox this should be fixed in `0.6.0`

**Comment by williamluke4** [comment]
Feel free to reopen if it is not fixed
